client/app/components/user/user-lang.controller.js

```diff
@@ -1,22 +0,0 @@
-angular.module("App").controller("UserLangueCtrl", [
-    "$scope",
-    "translator",
-
-    function ($scope, translator) {
-        "use strict";
-
-        // languages choice
-        $scope.selectedLanguage = translator.getSelectedAvailableLanguage();
-        $scope.availableLanguages = translator.getAvailableLanguages();
-
-        if (moment && moment.locale && $scope.selectedLanguage.value) {
-            moment.locale($scope.selectedLanguage.value.replace(/_/, "-"));
-        }
-
-        $scope.setSelectedLanguage = (newLanguage) => {
-            $scope.selectedLanguage = newLanguage;
-            localStorage.setItem("univers-selected-language", newLanguage.value);
```

Comment:
Why do we remove the localStorage call ?

Reply:
The whole file is deleted, it wasn't used at all

Reply:
The whole file is deleted, it wasn't used at all

Reply:
It's not my question :/

In others managers, we use univers-selected-language to determine the language at first loading.

https://github.com/ovh-ux/ovh-manager-telecom/blob/8dde0a6043aca2d83da1b85e726f29d93273b15a/client/app/app.js#L116

Why do we delete it in web manager ? :(

Reply:
This file wasn't used at all, I deleted it because it had no use.

The `univers-selected-language` processing is done by `app.js`:

    let defaultLanguage = "fr_FR";

    if (localStorage["univers-selected-language"]) {
        defaultLanguage = localStorage["univers-selected-language"];
    } else {
        localStorage["univers-selected-language"] = defaultLanguage;
    }

Reply:
And the language switching is done by the navbar:

    _(LANGUAGES)
                .filter((language) => _(language).has("name", "value"))
                .map((language) => ({
                    label: language.name,
                    click: () => {
                        localStorage["univers-selected-language"] = language.value;
                        window.location.reload();
                    },
                    lang: _(language.value).chain()
                        .words()
                        .head()
                        .value()
                }))